main.py

```diff
@@ -40,7 +41,12 @@
                 SidebarToggle.get_component(),
                 Content.get_component(),
             ],
-            style={"margin-right": "0", "margin-left": "0", "height": "100%"},
+            style={
+                "margin-right": "0",
```

Comment:
These margin left/right should be replaced with bootstrap classes:
`className="ml-0 mr-0",`